packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -131,6 +139,21 @@ class CupertinoButton extends StatefulWidget {
   /// Always defaults to [Alignment.center].
   final AlignmentGeometry alignment;
 
+  /// The color to use for the focus highlight for keyboard interactions.
+  ///
+  /// Defaults to a slightly transparent [color]. If [color] is null, defaults
+  /// to a slightly transparent [CupertinoColors.activeBlue].
+  final Color? focusColor;
+
+  /// {@macro flutter.widgets.Focus.focusNode}
+  final FocusNode? focusNode;
+
+  /// {@macro flutter.material.inkwell.onFocusChange}
```

Comment:
```suggestion
  /// {@macro flutter.material.inkwell.onFocusChange}
```

This references material lnkwell onFocusChange. When we just to document `onFocusChange` , unrelated to inkwell itself. I wonder other Cupertino widgets contain `onFocusChange` and how it's documented there. Otherwise we should have specific documentation here instead of InkWell reference.

Reply:
The only other place I saw it done was in CupertinoSwitch:
https://github.com/flutter/flutter/blob/d520f076477bf741ce9f44bd083730979ea182db/packages/flutter/lib/src/cupertino/switch.dart#L149

The macro is defined here:
https://github.com/flutter/flutter/blob/a7967703cd58b2164168950a92ccf30293d22bf1/packages/flutter/lib/src/material/ink_well.dart#L577-L582

Considering the definition is very generic, what the pros and cons are in reusing the macro here instead of duplicating?

Reply:
Couple of concerns regarding this.
1. InkWell reference comes from Material library,, if updated in the future it will have implications in the Cupertino library where InkWell cannot be used without `Material` widget surface. The references seems out of place.
2. If more Cupertino widgets are expected to use `onFocusChange` callback, we should have similar yet independent docs.

I suggest since this is using `FocusableActionDetector` from `widgets` library, we can make these doc reuse `FocusableActionDetector.onFocusChange` docs (turn into a template) or write original docs for the `CupertinoButton`.  I would lean towards writing original docs based on this discussion https://github.com/flutter/flutter/pull/142516#discussion_r1472065226 on a similar PR.